packages/styles/scss/components/card-link/index.scss

```diff
@@ -17,12 +17,21 @@
     text-decoration: none;
     display: flex;
     flex-direction: column;
-    padding: $spacing-05;
 
     > *:not(:last-child) {
       margin-bottom: $spacing-07;
     }
 
+    &__wrapper {
+      padding: $spacing-05;
+    }
+
+    &__image {
+      width: 100%;
+      height: auto;
+      object-fit: cover;
```

Comment:
> @photodow : yeah I am using object-fit: cover property in card link -image selector.

Oh nice. That's what I get for trying to go quick on this. Sorry about that. I guess I thought a wrapper was required to make it work, but to my surprise that's not the case.

Reply:
@photodow: I have addressed the above comments and updated the PR and let me know for ant changes.